### PR TITLE
fix error parameter with digits

### PR DESCRIPTION
**Pull request by Avi-Eyal**
fix Error logs thats their parameters contain digits

### CODE REVIEW

**Review by tzheleznyak — APPROVED** [review]
Looks good . Just add to the test plan a log check for this so we will do it in regression

File: app/domain/errors.rb [hunk 1]
```diff
@@ -446,12 +446,12 @@ module AuthnJwt
       )
 
       UnsupportedAuthenticator = ::Util::TrackableErrorClass.new(
-        msg: "Authenticator '{0-authenticator-name}' is unsupported.",
```
Comment (benoaviram) [inline app/domain/errors.rb:449]: @Avi-Eyal  what is the reason for this change?

Comment (Avi-Eyal) [reply app/domain/errors.rb:449]: There is no particular reason, the convention in the other logs is that where there is only one parameter or the specific parameter is self explained then you dont need to specify the parameter by name.

personally I dont really care to rollback my changes for this line

**Review by benoaviram — APPROVED** [review]
